test/libsolidity/smtCheckerTests/external_calls/call_with_value_1.sol

```diff
@@ -0,0 +1,14 @@
+contract C {
+	function g(address payable i) public {
+		require(address(this).balance == 100);
+		i.call{value: 10}("");
+		// Disabled due to Spacer nondeterminism
+		//assert(address(this).balance == 90); // should hold
+		// Disabled due to Spacer nondeterminism
+		//assert(address(this).balance == 100); // should fail
```

Comment:
Do we have some plan for dealing with this nondeterminism long-term?

Reply:
Maybe we could introduce some soltest setting for having a test skipped on some platforms? Would be better than having to skip it everywhere as we do now.

Reply:
Yea, could be too. I'd still vote for removing osx, from Earth.

Reply:
#11870